src/components/shared/Deeplinking.tsx

```diff
@@ -1,24 +1,37 @@
-import { NavigationActions } from "react-navigation";
+import { NavigationActions, NavigationParams } from "react-navigation";
 import url from "url";
 import { DEEPLINK_ROUTES } from "./Navigation";
 
 export function processDeeplink(link: string, navigation: any) {
+  console.log("YOLO", "Processing full deeplink " + link);
```

Comment:
#YOLO

Reply:
hahah we should probably remove these huh

Reply:
never!! just kidding yes